### PR TITLE
instructions to add ohAnalyzer to exist-db

### PR DESCRIPTION
**Pull request by marmoure**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+## Index The data using the custom Analyzer
+when creating the index config specify the `Analyzer` as `com.evolvedbinary.oh.OhAnalyzer`
+```xml
+<collection xmlns="http://exist-db.org/collection-config/1.0">
+    <index xmlns:wiki="http://exist-db.org/xquery/wiki" xmlns:html="http://www.w3.org/1999/xhtml" xmlns:atom="http://www.w3.org/2005/Atom">
+        <!-- Lucene index is configured below -->
+        <lucene>
+	        <analyzer class="com.evolvedbinary.oh.OhAnalyzer"/>
```
Comment (adamretter) [inline README.md:66]: There will now be two parameters that need to be configured:
1. `char[] punctuationDictionary`
2. `int minimumTermLength`

**Review by adamretter — CHANGES_REQUESTED** [review]
We need to add in the parameters, but otherwise this looks good.

File: README.md (hunk 1 above)
Comment (marmoure) [reply README.md]: can you verify the types please

Comment (adamretter) [reply README.md]: Perfect :-)

**Pull request merged**